video/fmt-conversion.h

```diff
@@ -20,6 +20,9 @@
 
 #include <libavutil/pixfmt.h>
 
+#define AV_PIX_FMT_RPI4_8 199
+#define AV_PIX_FMT_RPI4_10 200
+
```

Comment:
This is where it isn't going to be mergeable. We need to wait until it's in upstream ffmpeg (which I have no confidence in happening, unless it's you who's doing the work to make that happen). The upstream pix fmt assignments can change over time and then things silently break.

Reply:
I had a feeling this was going to be problematic. I tried to find a #define created by rpi-ffmpeg that would work but couldn't. I can ask over there in the meantime.

Reply:
Latest commit should resolve this.